package-lock.json

```diff
@@ -1,4597 +1,8 @@
 {
   "name": "@octokit/oauth-authorization-url",
   "version": "0.0.0-development",
-  "lockfileVersion": 2,
```

Comment:
Looks like you went with a version of npm that's a little too old here

Reply:
I'll update; one moment!

Reply:
I've regenerated it with Node v18. Please let me know if you'd like to see other changes!

Reply:
:sweat_smile: That is too new. If you set `lockfileVersion` to 2 manually and regenerate, it should be good

Reply:
Are you sure? I believe this repo has already dropped support for Node v16 and v16 [here](https://github.com/octokit/oauth-authorization-url.js/commit/3a443adc2c78810c9817e77c0f4430941539aa0f).

Reply:
Actually, yeah it should be fine